packages/node/rollup.config.js

```diff
@@ -63,6 +64,13 @@ export default [
         namedExports: {
           "../../node_modules/typescript-memoize/dist/memoize-decorator.js": [
             "Memoize"
+          ],
+          "../../node_modules/rpc-server/dist/index.js": [
+            "jsonRpcMethod",
+            "jsonApiOperation",
+            "jsonApiType",
```

Comment:
the `jsonApi*` exports can probably be omitted?

Reply:
Sure thing.